core/ir/aarch64/codec.txt

```diff
@@ -1119,6 +1119,19 @@ x101101011000000000101xxxxxxxxxx  cls     wx0 : wx5
 0001111011100010010000xxxxxxxxxx     fcvt      s0 : h5
```

Comment:
Please see https://dynamorio.org/page_code_reviews.html#sec_commit_messages for commit message conventions.

> Add support for FCVTZU to DynamoRIO codec

The title needs a prefix with the issue number and since this is one of multiple a short description.

>
> FCVTZU was overlooked in the original AArch64 port.
> This is the first in a series of patches to implement missing instructions in the codec.
> 
> This patch is related to issue 2626:
> #2626

This should follow the convention: `Issue: #2626`.

Reply:
Please reply "Done" or somesuch to each comment.  This is documented in the code review workflow: https://dynamorio.org/page_code_reviews.html#autotoc_md113  Please look over the code commit and review documentation.